core-build/gulp-core-build/src/tasks/JestReporter.ts

```diff
@@ -1,3 +1,8 @@
+import { FileSystem } from '@microsoft/node-core-library';
+import * as path from 'path';
+import * as xml from 'xml';
+import * as Jest from 'jest-cli';
+import TestResults = require('jest-nunit-reporter/src/Testresults');
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
 const DEFAULT_REPORTER: any = require('jest-cli/build/reporters/default_reporter').default;
```

Comment:
Does this have types? Can we replace this (and the `extends` on line 14) with a more conventional extends?

We should be able to import this with

```typescript
import { default as DEFAULT_REPORTER} from 'jest-cli/build/reporters/default_reporter';
```

Reply:
Unfortunately no types. But the import change works. Gotta get used to how that stuff is formatted heh.